data/SugarBean.php

```diff
@@ -3465,7 +3465,9 @@ public function get_list(
             '',
             false,
             null,
-            $singleSelect
+            $singleSelect,
+            false,
+            'LIMIT ' . $row_offset . ', ' . $limit
```

Comment:
Use of the comment syntax may help with comparability
https://dev.mysql.com/doc/refman/8.0/en/comments.html

`'/*! LIMIT ' . $row_offset . ', ' . $limit . ' */' `